src/NamespacedPathStackResolver.php

```diff
@@ -60,12 +60,10 @@ public function __construct($options = null)
      * Add a path to the stack with the given namespace.
      *
      * @param string $path
-     * @param string $namespace
-     * @return void
      * @throws ViewException\InvalidArgumentException for an invalid path
      * @throws ViewException\InvalidArgumentException for an invalid namespace
      */
-    public function addPath($path, $namespace = self::DEFAULT_NAMESPACE)
+    public function addPath($path, ?string $namespace = self::DEFAULT_NAMESPACE) : void
```

Comment:
I see const defined in this file, can we add also visibility on const definition please?

Reply:
Sure.